test/registries.test.js

```diff
@@ -0,0 +1,48 @@
+describe('Registry', function () {
+  describe('one per app', function () {
+    it('should allow two apps to reuse the same model name', function (done) {
+      var appFoo = loopback();
+      var appBar = loopback();
+      var modelName = 'MyModel';
+      var subModelName = 'Sub' + modelName;
+      var settings = {base: 'PersistedModel'};
+      appFoo.set('perAppRegistries', true);
+      appBar.set('perAppRegistries', true);
```

Comment:
I'm deleting this comment, the PR description says that the flag is not implemented yet.